deps/wazuh_testing/wazuh_testing/end_to_end/remote_operations_handler.py

```diff
@@ -12,209 +12,298 @@
-                                                                          host,
-                                                                          package_data_to_use,
-                                                                          current_datetime)
-
-        if len(results['evidences'].get('alerts_not_found_from', [])) > 0 or \
-                len(results['evidences'].get('alerts_not_found_to', [])) > 0 or \
-                len(results['evidences'].get('alerts_not_found', [])) > 0:
-            results['checks']['all_successfull'] = False
-
-    # Check unexpected states
-    if 'no_indices' in check_data and check_data['no_indices']:
-        logging.info(f'Checking vulnerability state index for {host}')
-        results['evidences']["states_found_unexpected"] = vulnerability_index
-
-        if len(results['evidences']['states_found_unexpected']) > 0:
-            results['checks']['all_successfull'] = False
-
-    elif check_data['state_index']:
-        if operation == 'update' or operation == 'remove':
-            evidence_key = 'states_found_unexpected_from' if operation == 'update' else 'states_found_unexpected'
-            package_data_to_use = package_data['from'] if operation == 'update' else package_data
-            # Check states from previous package are mitigated
-            results['evidences'][evidence_key] = check_vuln_state_index(host_manager, host, package_data_to_use,
-                                                                        current_datetime)
-            if len(results['evidences'][evidence_key]) != len(package_data_to_use['CVE']):
-                results['checks']['all_successfull'] = False
-
-        elif operation == 'install' or operation == 'update':
-            # Check states from new package are found
-            evidence_key = 'states_not_found_to' if operation == 'update' else 'states_not_found'
-            package_data_to_use = package_data['to'] if operation == 'update' else package_data
-            results['evidences'][evidence_key] = check_vuln_state_index(host_manager, host, package_data_to_use,
-                                                                        current_datetime)
-
-            if len(results['evidences'][evidence_key]) != 0:
-                results['checks']['all_successfull'] = False
-
-
-def install_package(host: str, operation_data: Dict[str, Dict], host_manager: HostManager):
-    """
-    Install a package on the specified host.
 
-    Args:
-        host (str): The target host on which to perform the operation.
-        operation_data (dict): Dictionary containing operation details.
-        host_manager (HostManager): An instance of the HostManager class containing information about hosts.
 
-    Raises:
-        ValueError: If the specified operation is not recognized.
+def get_vulnerabilities_not_found(
+    vulnerabilities_found: List, expected_vulnerabilities: List
+) -> List:
```

Comment:
```suggestion
) -> list:
```

Reply:
We are using the typing module. I suggest maintain this along all the framework